client/templates/initiative/edit/initiativeEdit.js

```diff
@@ -0,0 +1,61 @@
+Template.initiativeEditFull.onCreated(function initiativeEditFullonCreated() {
+  Session.set('initiativeActive', Template.currentData().initiative.active);
+});
+
+Template.initiativeEditFull.onRendered(function initiativeEditFullonRendered() {
+  this.$('textarea#summary').characterCounter();
+  this.$('select#category_select').material_select();
+});
+
+
+// initiativeEditFull Helpers
+Template.initiativeEditFull.helpers({
+  getCategories: function getCategories() {
+    return InitiativeCategories;
+  },
+  getSelected: function getSelected(category, compare) {
+    return category === compare ? 'selected' : '';
+  }
+});
+
+// initiativeEditFull events
+Template.initiativeEditFull.events({
+  'submit form': function submitEditForm(ev, tpl) {
+    ev.preventDefault();
+    updateInitiative(this, tpl);
+  },
+  'click .switch label, click .switch input': function clickActiveInactive() {
+    Session.set('initiativeActive', !Session.get('initiativeActive'));
+  }
+});
+
+function updateInitiative(initiative, tpl) {
```

Comment:
This needs to be a function expression and needs to be moved up to the top since its a var declaration

`var updateInitiative = function updateInitiative(initiative, tpl) {`